src/components/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,147 @@
+﻿import { useState, useEffect, useRef, FunctionComponent } from 'react'
+import classNames from 'classnames'
+import styles from '@/components/Textarea/Textarea.module.scss'
+
+type TextareaProps = React.PropsWithChildren<{
+  value?: string;
+  maxlength?: number;
+  maxlengthDesc?: string;
+  title?: string;
+  placeholder?: string;
+  description?: string;
+  error?: string;
+  disabled?: boolean;
+  rows?: "auto" | number;
+  onChange?: (value: string) => void;
+  required?: boolean;
+  wrapperClassName?: string;
+  labelClassName?: string;
+  descriptionClassName?: string;
+  errorClassName?: string;
+}>
+
+type HandlerEvent = <K extends keyof HTMLElementEventMap>(this: HTMLTextAreaElement, ev: HTMLElementEventMap[K]) => any
+
+const Textarea: FunctionComponent<TextareaProps> = ({
+  value: propsValue,
+  onChange,
+  maxlength,
+  maxlengthDesc,
+  title,
+  placeholder,
+  description,
+  error,
+  disabled,
+  rows,
+  required,
+  wrapperClassName,
+  labelClassName,
+  descriptionClassName,
+  errorClassName,
+}) => {
+
+  const textarea = useRef(null)
+  const [value, setValue] = useState(propsValue || '')
```

Comment:
Please change the component to be uncontrolled. So remove the value state and only let use only onChange and value from the props.